src/components/ControlPane.js

```diff
@@ -6,11 +6,13 @@ export default class ControlPane extends React.Component {
   controlFor(field) {
     const { entry, getMedia, onChange, onAddMedia, onRemoveMedia } = this.props;
     const widget = resolveWidget(field.get('widget'));
+    const value = entry.getIn(['data', field.get('name')]);
+    if (!value) return null;
```

Comment:
This can be a false positive if `value === 0` or `value ===''`. Since `value` isn't `Boolean` I'd explicitely check if it isnt `undefined` or `null` here.